src/internal/streamstore/streamstore.go

```diff
@@ -80,6 +81,7 @@ func (ss *streamStore) WriteBackupDetails(
 		ss.service,
 		nil,
 		nil,
+		isIncremental,
```

Comment:
this depends on how much of the underlying incremental behavior we want to expose to higher layer. For now since we're already passing `nil` for the set of bases I think it's fine to just hardcode it to `false` and not even expose it as another parameter

Reply:
ah, I was looking at the wrong param.  You're right, no point in the flag if the input is nil.